share/shwap/row.go

```diff
@@ -183,3 +183,25 @@ func sideFromProto(side pb.Row_HalfSide) RowSide {
 	}
 	return Right
 }
+
+func (s RowSide) toString() string {
```

Comment:
If we make it public String(), then loggers and stuff would recognise it as a stringer and print it instead, which is handy